From b228310e8dfd490dd7721f7e1e91fe07e4a5ecd4 Mon Sep 17 00:00:00 2001
From: Dmitrijs Ledkovs <xnox@debian.org>
Date: Sun, 25 Aug 2013 12:43:52 +0200
Subject: [PATCH] Fix pybuild.pm when PYBUILD_INTERPRETERS is set

somehow I missed this case in a617d883f519e924f487cc87f956f1bbaa1735da commit - Piotr
---
 dh/pybuild.pm    | 2 +-
 2 files changed, 9 insertions(+), 1 deletion(-)

diff --git a/dh/pybuild.pm b/dh/pybuild.pm
index 302d712..1f9c3b9 100644
--- a/dh/pybuild.pm
+++ b/dh/pybuild.pm
@@ -88,7 +88,7 @@ sub pybuild_commands {
 	my @result;
 
 	if ($ENV{'PYBUILD_INTERPRETERS'}) {
-		push @result, "pybuild --$step @options";
+		push @result, ['pybuild', "--$step", @options];
 	}
 	else {
 		# get interpreter packages from Build-Depends{,-Indep}:
-- 
1.8.4.rc3

